internals/models.py

```diff
@@ -972,113 +974,135 @@ def put(self, notify=True, **kwargs):
+  feature_type = ndb.IntegerProperty(default=FEATURE_TYPE_INCUBATE_ID)
+  intent_stage = ndb.IntegerProperty(default=0)
+  summary = ndb.StringProperty(required=True)
+  unlisted = ndb.BooleanProperty(default=False)
   # TODO(jrobbins): Add an entry_state enum to track app-specific lifecycle
   # info for a feature entry as distinct from process-specific stage.
-  deleted = db.BooleanProperty(default=False)
-  motivation = db.StringProperty(multiline=True)
-  star_count = db.IntegerProperty(default=0)
-  search_tags = db.StringListProperty()
-  comments = db.StringProperty(multiline=True)
-  owner = db.ListProperty(db.Email)
-  footprint = db.IntegerProperty()  # Deprecated
+  deleted = ndb.BooleanProperty(default=False)
+  motivation = ndb.StringProperty()
+  star_count = ndb.IntegerProperty(default=0)
+  search_tags = ndb.StringProperty(repeated=True)
+  comments = ndb.StringProperty()
+  # owner = db.ListProperty(db.Email)
+  owner = ndb.StringProperty(repeated=True)
+  footprint = ndb.IntegerProperty()  # Deprecated
 
   # Tracability to intent discussion threads
-  intent_to_implement_url = db.LinkProperty()
-  intent_to_ship_url = db.LinkProperty()
-  ready_for_trial_url = db.LinkProperty()
-  intent_to_experiment_url = db.LinkProperty()
-  i2e_lgtms = db.ListProperty(db.Email)  # Currently, only one is needed.
-  i2s_lgtms = db.ListProperty(db.Email)
+  # intent_to_implement_url = db.LinkProperty()
+  intent_to_implement_url = ndb.StringProperty()
+  # intent_to_ship_url = db.LinkProperty()
+  # ready_for_trial_url = db.LinkProperty()
+  # intent_to_experiment_url = db.LinkProperty()
+  # i2e_lgtms = db.ListProperty(db.Email)  # Currently, only one is needed.
+  # i2s_lgtms = db.ListProperty(db.Email)
+  intent_to_ship_url = ndb.StringProperty()
+  ready_for_trial_url = ndb.StringProperty()
+  intent_to_experiment_url = ndb.StringProperty()
+  i2e_lgtms = ndb.StringProperty(repeated=True)  # Currently, only one is needed.
+  i2s_lgtms = ndb.StringProperty(repeated=True)
 
   # Chromium details.
-  bug_url = db.LinkProperty()
-  launch_bug_url = db.LinkProperty()
-  initial_public_proposal_url = db.LinkProperty()
-  blink_components = db.StringListProperty(required=True, default=[BlinkComponent.DEFAULT_COMPONENT])
-  devrel = db.ListProperty(db.Email)
-
-  impl_status_chrome = db.IntegerProperty(required=True)
-  shipped_milestone = db.IntegerProperty()
-  shipped_android_milestone = db.IntegerProperty()
-  shipped_ios_milestone = db.IntegerProperty()
-  shipped_webview_milestone = db.IntegerProperty()
+  # bug_url = db.LinkProperty()
+  # launch_bug_url = db.LinkProperty()
+  # initial_public_proposal_url = db.LinkProperty()
+  bug_url = ndb.StringProperty()
+  launch_bug_url = ndb.StringProperty()
+  initial_public_proposal_url = ndb.StringProperty()
+  # blink_components = db.StringListProperty(required=True, default=[BlinkComponent.DEFAULT_COMPONENT])
+  blink_components = ndb.StringProperty(repeated=True)
```

Comment:
`StringListProperty()` has been deprecated. The replacement is similar to that of `ListProperty()`. However, the arguments `repeated` and `default/required` cannot be used together. As this [link](https://googleapis.dev/python/python-ndb/latest/model.html) says, 

> The repeated and required/default options are mutually exclusive: a repeated property cannot be required nor can it specify a default value (the default is always an empty list and an empty list is always an allowed value), but a required property can have a default.

Reply:
Okay. I will try to do it and let you know if I face any problem.

Reply:
As it turns out, other than `*_tests.py`, we create a feature only in `guide.py` while processing form data. There, we already provide a value to the `blink_components` property. I will update it in the tests and let you know.

Reply:
I am using the constructor for assinging the default value. See [this](https://github.com/GoogleChrome/chromium-dashboard/pull/1352/commits/d67262c4b566b7cce2f6235ddb069ccb005fd57c) commit.

Reply:
@jrobbins, please mark this as resolved once you have reviewed the commit mentioned in the above comment.